src/bci_build/package/kiwi.py

```diff
@@ -47,6 +48,7 @@
             "timezone",
             "xfsprogs",
         ],
+        custom_end=f"{generate_package_version_check('python3-kiwi', kiwi_ver, 'minor')}",
```

Comment:
this and a reformat:
```suggestion
        custom_end=f"{generate_package_version_check('python3-kiwi', to_major_minor_version(kiwi_ver), 'minor')}",
```